packages/tools/viewer-alpha/src/viewerFactory.ts

```diff
@@ -1,4 +1,5 @@
-import { Engine, type EngineOptions } from "core/Engines";
+import { Engine } from "core/Engines/engine";

```

Comment:
@RaananW similar issue again here, where `import { Engine } from "core/Engines";` was becoming `import { Engine } from "@babylonjs/core/engines.js"`.

Reply:
we don't import from directories for this reason. always import directly from the right file (in our monorepo)

Reply:
Also, small nit (and feel free to ignore this):

`import { type EngineOptions } from ....` is only supported from typescript 4.9 and up (the "type" being inside the {} and not outside). This means that setting it this way (which is totally correct syntax) will prevent people with ts < 4.9 to use the lib.